### PR TITLE
fix: accept space in fixed location

### PR DESCRIPTION
**Pull request by tuanha168**
I add some small tweak so that fixed localtion can accept string with space like `"New York"`

### CODE REVIEW

File: scripts/dracula.sh [hunk 1]
```diff
@@ -180,7 +180,7 @@ main()
 
     elif [ $plugin = "weather" ]; then
       IFS=' ' read -r -a colors <<< $(get_tmux_option "@dracula-weather-colors" "orange dark_gray")
-      script="#($current_dir/weather_wrapper.sh $show_fahrenheit $show_location $fixed_location)"
+      script="#($current_dir/weather_wrapper.sh $show_fahrenheit $show_location "\"$fixed_location\"")"
```
Comment (ethancedwards8) [inline scripts/dracula.sh:183]: Couldn't this just be `"${fixed_location}"`? https://stackoverflow.com/questions/8748831/when-do-we-need-curly-braces-around-shell-variables

File: scripts/weather_wrapper.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ main()
 
   if [ "$(expr ${TIME_LAST} + ${RUN_EACH})" -lt "${TIME_NOW}" ]; then
     # Run weather script here
-    $current_dir/weather.sh $fahrenheit $location $fixedlocation > "${DATAFILE}"
+    $current_dir/weather.sh $fahrenheit $location "$fixedlocation" > "${DATAFILE}"
```
Comment (ethancedwards8) [inline scripts/weather_wrapper.sh:23]: Couldn't this just be `"${fixed_location}"`? https://stackoverflow.com/questions/8748831/when-do-we-need-curly-braces-around-shell-variables

File: scripts/dracula.sh (hunk 1 above)
Comment (tuanha168) [reply scripts/dracula.sh:183]: I tried `"${fixed_location}"` but no luck, still need escape quote when have space in like this in `.tmux.conf`
`set -g @dracula-fixed-location "New York"`

Just `New` got inside and the weather also wrong
image

File: scripts/weather_wrapper.sh (hunk 2 above)
Comment (tuanha168) [reply scripts/weather_wrapper.sh:23]: (duplicate comment maybe ?)

yes, it's the same with my changes

**Comment by tuanha168** [comment]
@ethancedwards8 
any update ?

**Pull request closed without merging**